tests/all-spark-notebook/test_spark_notebooks.py

```diff
@@ -5,7 +5,8 @@
 from pathlib import Path
 
 import pytest  # type: ignore
-from conftest import TrackedContainer
```

Comment:
When I open PyCharm at the root of this project, it tries to resolve this import.
Because there is no __init__.py, it tries to import from the root of the repo and fails to find this import.

Reply:
I think it worked, because pytest might do something with sys.path.
https://docs.pytest.org/en/6.2.x/pythonpath.html#standalone-test-modules-conftest-py-files